### ISSUE TITLE
choose custom hyperparameters ?

### ISSUE THREAD

**Issue by TerrafYassin** [ZhAnGToNG1/MSFC-Net#16]
please , how to add my custom hyperparameters ( learning rate , optimizer ..) ?

**Comment by ZhAnGToNG1** [comment]
See https://github.com/ZhAnGToNG1/MSFC-Net/blob/master/src/lib/opts.py

**Comment by TerrafYassin** [comment]
there is no option to change optimizer in opts.py

**Comment by TerrafYassin** [comment]
and please , shold i enable --trainval to validate model while training ? 
in your experiment you have enabled  --trainval  ?

**Comment by ZhAnGToNG1** [comment]
Change optimizer in https://github.com/ZhAnGToNG1/MSFC-Net/blob/master/src/train.py#L34.
yes, enable --trainval to validate model by evluating loss.

**Comment by TerrafYassin** [comment]
thank you
